x-pack/plugins/fleet/server/services/agents/action_status.ts

```diff
@@ -39,6 +39,11 @@ export async function getActionStatuses(
           terms: { field: 'action_id', size: actions.length || 10 },
           aggs: {
             max_timestamp: { max: { field: '@timestamp' } },
+            agent_count: {
+              cardinality: {
+                field: 'agent_id',
```

Comment:
Should we use `precision_threshold` here to have a more accurate response https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-cardinality-aggregation.html ?